src/openrct2/ride/ride.h

```diff
@@ -80,16 +80,26 @@ typedef struct vehicle_colour_preset_list {
 } vehicle_colour_preset_list;
 assert_struct_size(vehicle_colour_preset_list, (1 + 256 * 3));
 
+typedef struct rct_ride_name {
+    rct_string_id name;
+    rct_string_id description;
+} rct_ride_name;
+
```

Comment:
Yes, it is 4 bytes, but the code was moved in from a non-packed area to a packed one, so it would be better to keep it outside. You can check if the places where `rct_ride_name` is used put any constraints on size or layout of it.